client.go

```diff
@@ -377,22 +378,23 @@ func (c *Client) PutObject(path string, data interface{}) (io.ReadCloser, error)
 	return resp, nil
 }
 
-//Put will send a PUT request through the client
+// Put will send a PUT request through the client
 func (c *Client) Put(path string, body io.Reader) (io.ReadCloser, int, error) {
-	return c.send("PUT", path, body)
+	h := map[string]string{"Content-Type": "application/json"}
```

Comment:
can you move this up into PutObj and PostObj respectively?  Technically those are the only places where we _know_ that we sending JSON.  (Though yes, this entire service is JSON right now, but it _could_ change in the future

Reply:
yeah that sounds good :)